jbpm-db-scripts/src/test/java/org/jbpm/persistence/scripts/GenerateDDLScriptsTests.java

```diff
@@ -36,33 +35,38 @@
  * Utility class for generating DDL scripts (create and drop) please ignore it.
  */
 @RunWith(Parameterized.class)
-@Ignore
```

Comment:
Why is removed? I think this is not exactly a Test class, as there are no assertions.
We could expand this PR [jbpm#1714](https://github.com/kiegroup/jbpm/pull/1714) to check also new files at certification matrix only for postgres allocated dbs.

Reply:
done